### ISSUE TITLE
Package is compiled for arm64

### ISSUE THREAD

**Issue by nonnymoose** [termux/play-audio#3]
So I cannot run it on my armv7l device:
```
u0_a159@localhost:~$ play-audio Word\ Crimes.mp3
/data/data/com.termux/files/usr/libexec/play-audio: 1: /data/data/com.termux/files/usr/libexec/play-audio: ELF��: not found
/data/data/com.termux/files/usr/libexec/play-audio: 4: /data/data/com.termux/files/usr/libexec/play-audio: Syntax error: "(" unexpected
u0_a159@localhost:~$ $PREFIX/libexec/play-audio
bash: /data/data/com.termux/files/usr/libexec/play-audio: cannot execute binary file: Exec format error
u0_a159@localhost:~$ file !!
file $PREFIX/libexec/play-audio
/data/data/com.termux/files/usr/libexec/play-audio: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /system/bin/linker64, not stripped
u0_a159@localhost:~$
```

**Comment by fornwall** [comment]
Thanks a lot for reporting! This has been fixed in the 0.5 update of the package which can now be installed.

**Comment by nonnymoose** [comment]
Awesome, thanks! It now works on my device.
